src/features/goblins/storageHouse/components/DeliverItems.tsx

```diff
@@ -95,18 +83,23 @@ export const DeliverItems: React.FC<Props> = ({ onWithdraw }) => {
     load();
   }, []);
 
-  useEffect(() => {
-    // Only grab the deliverable items
-    const resourceInventory = getDeliverableItems(
-      goblinState.context.state.inventory
+  const hasWrongInputs = (): boolean => {
+    const entries = Object.entries(selected) as [InventoryItemName, Decimal][];
+    const wrongInputs = entries.filter(
+      ([k, v]) => v?.lte(0) || v?.gt(inventory[k] || new Decimal(0))
```

Comment:
Personally, I am not a fan of single char variable names. I get that in this instance it is representing key and value but would love to aim to always have descriptive naming. It just makes readability so much better.

Reply:
Fixed